osx/post_installation.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+
+echo "Now preparing KA-Lite dependencies.."
+
+SCRIPTPATH=$( cd $(dirname $0) ; pwd -P )
+
+PYRUN_NAME="pyrun-2.7"
+PYRUN_DIR="/Users/Shared/$PYRUN_NAME"
+PYRUN="$PYRUN_DIR/bin/pyrun"
+PYRUN_PIP="$PYRUN_DIR/bin/pip"
+BIN_PATH="$PYRUN_DIR/bin"
+ASSESSMENT_SRC="/Users/Shared/assessment/assessment.zip"
+SHEBANGCHECK_PATH="/Users/Shared/scripts/"
+
+SYMLINK_FILE="/Users/Shared/pyrun-2.7/bin/kalite"
+SYMLINK_TO="/usr/local/bin"
+COMMAND_SYMLINK="ln -s $SYMLINK_FILE $SYMLINK_TO"
+
+TMP="/tmp/"
+PLIST=".plist"
+ORG="org.learningequality.kalite"
+TARGET="/Library/LaunchAgents/"
+KALITE=$(which kalite)
+
+# Symlink kalite executable to /usr/local/bin
+if [ -f "$KALITE" ]; then
+    echo "$KALITE is already exist."
+else
+    $COMMAND_SYMLINK
+fi
+
+function update_env {
+    echo "Updating KALITE_PYTHON environment variable"
+    launchctl setenv  KALITE_PYTHON "$PYRUN"
+    export KALITE_PYTHON="$PYRUN"
+    if [ $? -ne 0 ]; then
+        echo ".. Abort!  Exporting KALITE_PYTHON '$KA_LITE_APP_PATH'."
+        exit 1
+    fi
+}
+
+if [ -z ${KALITE_PYTHON+0} ]; then
+    update_env
+else
+    if [ "KALITE_PYTHON" != $PYRUN ]; then
+        update_env
+    fi
+fi
```

Comment:
Let's just run `update_env()` here, no need to check if it's value is the same or not.